.pipelines/templates/soak-test-template.yml

```diff
@@ -0,0 +1,36 @@
+jobs:
+  - job: soak_test_aks_engine
+    timeoutInMinutes: 10
+    cancelTimeoutInMinutes: 5
+
+    workspace:
+      clean: all
+
+    variables:
+      - group: kubernetes-kms-soak-aks-engine
+
+    steps:
+      - script: make install-soak-prerequisites
+        displayName: "Install e2e soak test prerequisites"
+
+      - script: |
+          key=$(cat <<-END
+          -----BEGIN RSA PRIVATE KEY-----
+          $(CLIENT_KEY)
+          -----END RSA PRIVATE KEY-----
+          END
+          )
+
+          kubectl config set-cluster kms-aks-engine --server=$(API_SERVER_ENDPOINT)
+          kubectl config set clusters.kms-aks-engine.certificate-authority-data $(CA_DATA)
+          kubectl config set-credentials cluster-admin
+          kubectl config set users.cluster-admin.client-certificate-data $(CLIENT_CERTIFICATE_DATA)
+          kubectl config set users.cluster-admin.client-key-data "$(echo "$key" | base64 -i -)"
+          kubectl config set-context kms --user=cluster-admin --cluster=kms-aks-engine
+          kubectl config use-context kms
```

Comment:
We can store the entire kubeconfig as a secure file and download as part of the pipeline. We do this today in `aad-pod-identity`.

https://github.com/Azure/aad-pod-identity/blob/master/.pipelines/templates/soak-test.yml#L34-L43

Reply:
done